main_test.go

```diff
@@ -284,6 +284,24 @@ func TestServe(t *testing.T) {
 			},
 			startHTTP,
 		},
+		{
+			"http POST request with session id",
+			"testdata/http-session-id.yml",
+			func(t *testing.T) {
+				req, err := http.NewRequest("POST", "http://127.0.0.1:9090/?query_id=45395792-a432-4b92-8cc9-536c14e1e1a9&extremes=0&session_id=default-session-id233", bytes.NewBufferString("SELECT * FROM system.numbers LIMIT 10"))
```

Comment:
nit: would be nice to keep the line length consistent with the rest of the code.

Reply:
Adjusted